cockroachcloud/migrations-page.md

```diff
@@ -83,23 +78,87 @@ The dump file must be smaller than 4 MB. `INSERT` and `COPY` statements will be
-| Errors         | The number of SQL errors preventing a migration from attaining `READY TO FINALIZE` status. |
+<section class="filter-content" markdown="1" data-scope="postgres">
+To generate an appropriate PostgreSQL schema file, run the [`pg_dump` utility](https://www.postgresql.org/docs/current/app-pgdump.html) and specify the `-s` or `--schema-only` options to extract **only the schema** of a PostgreSQL database to a `.sql` file.
+</section>
 
-To view the [**Summary Report**](#summary-report) or [**Statements** list](#statements-list) for a migration, click the migration name.
+<section class="filter-content" markdown="1" data-scope="mysql">
+To generate an appropriate MySQL schema file, run the [`mysqldump` utility](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) and specify the `-d` or `--no-data` options to extract **only the schema** of the MySQL database to a `.sql` file.
+</section>
+
+<section class="filter-content" markdown="1" data-scope="oracle">
+To generate an appropriate Oracle schema file, run the [`expdp` utility](https://docs.oracle.com/cd/E11882_01/server.112/e22490/dp_export.htm#i1007509) to extract **only the schema** of the Oracle database to a `.sql` file.
+</section>
+
+<section class="filter-content" markdown="1" data-scope="mssql">
+To generate an appropriate Microsoft SQL Server schema file, use either [SQL Server Management Studio](https://learn.microsoft.com/en-us/sql/ssms/tutorials/scripting-ssms?view=sql-server-ver16#script-a-database-by-using-the-script-option) or the equivalent [`mssql-scripter` utility](https://github.com/microsoft/mssql-scripter) to extract **only the schema** of the SQL Server database to a `.sql` file.
+</section>
+
+The dump file must be smaller than 4 MB. `INSERT` and `COPY` statements will be ignored in schema conversion. To add a schema file:
+
+1. In step 2 of the **Add SQL Schema** dialog, click **Upload File**. Click the upload box and select a `.sql` file, or drop a `.sql` file directly into the box.
+1. Click **Convert** and wait for the schema to be analyzed. A loading screen is displayed. Depending on the size and complexity of the SQL dump, analyzing the schema can require up to several minutes.
+1. When analysis is complete, review the [**Summary Report**](#summary-report) and edit, add, or remove SQL statements in the [**Statements** list](#statements-list).
+
+<section class="filter-content" markdown="1" data-scope="postgres mysql">
+#### Use Credentials
+
+The Schema Conversion Tool can connect directly to a PostgreSQL or MySQL database to obtain the schema. To add a schema using credentials:
+
+1. In step 2 of the **Add SQL Schema** dialog, click **Use Credential**. Select the credentials to use. If the list is empty, you can [add credentials](#add-database-credentials).
```

Comment:
With the change to only show the credentials that exist for the selected dialect, should we call that out here in the docs as to why?

Reply:
Done.